system/Debug/BaseExceptionHandler.php

```diff
@@ -0,0 +1,248 @@
+<?php
+
+/**
+ * This file is part of CodeIgniter 4 framework.
+ *
+ * (c) CodeIgniter Foundation <admin@codeigniter.com>
+ *
+ * For the full copyright and license information, please view
+ * the LICENSE file that was distributed with this source code.
+ */
+
+namespace CodeIgniter\Debug;
+
+use CodeIgniter\API\ResponseTrait;
+use CodeIgniter\HTTP\RequestInterface;
+use CodeIgniter\HTTP\ResponseInterface;
+use Config\Exceptions as ExceptionsConfig;
+use Throwable;
+
+/**
+ * Provides common functions for exception handlers,
+ * especially around displaying the output.
+ */
+abstract class BaseExceptionHandler
+{
+    use ResponseTrait;
```

Comment:
What is needed from this trait? I find it hard to believe our exception handler needs all those methods. If it is just 1-2 I'd rather see them duplicated.

Reply:
This PR is based on Lonnie's one.

Reply:
Moved to the child class.